### PR TITLE
Meetings

### PR DESCRIPTION
**Pull request by hbjorgo**
Added Outlook meeting integration. The NuGet package targets netcoreapp3.1, so the projects had to be retargeted from netstandard2.1.

### CODE REVIEW

File: src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/20200929073210_InitialInvitation.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.Migrations
+{
+    public partial class InitialInvitation : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "CommPkg",
```
Comment (llykkelarsen) [inline src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/20200929073210_InitialInvitation.cs:11]: should table names all be plural?

File: src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        [HttpPost]
+        public async Task<ActionResult<int>> CreateInvitation(
+            [FromHeader(Name = CurrentPlantMiddleware.PlantHeader)]
+            [Required]
+            string plant,
+            [FromBody] CreateInvitationDto dto)
+        {
+            var result = await _mediator.Send(
+                new CreateInvitationCommand(
+                    new CreateMeetingCommand(dto.Meeting.Title, dto.Meeting.BodyHtml, dto.Meeting.Location, dto.Meeting.StartTime, dto.Meeting.EndTime, dto.Meeting.ParticipantOids)));
```
Comment (llykkelarsen) [inline src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs:41]: wrap long line

File: src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            [FromHeader(Name = CurrentPlantMiddleware.PlantHeader)]
+            [Required]
+            string plant,
+            [FromRoute] int id,
+            [FromBody] EditInvitationDto dto)
+        {
+            var result = await _mediator.Send(
+                new EditInvitationCommand(
+                    id,
+                    new EditMeetingCommand(dto.Meeting.Title, dto.Meeting.BodyHtml, dto.Meeting.Location, dto.Meeting.StartTime, dto.Meeting.EndTime, dto.Meeting.ParticipantOids)));
```
Comment (llykkelarsen) [inline src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs:56]: wrap long line

File: src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+{
+    [ApiController]
+    [Route("Invitations")]
+    public class InvitationsController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public InvitationsController(IMediator mediator) => _mediator = mediator;
+
+        [HttpGet("{id}")]
```
Comment (llykkelarsen) [inline src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs:21]: Permissions? Should anyone be able to do this?

**Review by llykkelarsen — APPROVED** [review]
Nice work getting the tests done:-)

File: src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs (hunk 4 above)
Comment (hbjorgo) [reply src/Equinor.ProCoSys.IPO.WebApi/Controllers/Invitation/InvitationsController.cs:22]: This isn't ready yet. Added TODOs.

**Pull request merged**